mathbot/modules/latex/__init__.py

```diff
@@ -28,7 +28,7 @@
 
 LATEX_SERVER_URL = 'http://rtex.probablyaweb.site/api/v2'
 DELETE_EMOJI = '🗑'
-
+SPOILER_REGEXP = re.compile(r'\|\|\s+.*\s+\|\|$')
```

Comment:
It seems that it checks if `||` is somewhere in the latex and if the latex ends with `||`. Shouldn't it check if it starts with those? like `r'^\|\|\s+.*\s+\|\|$'`

Reply:
I didn't realize that it was parsed out to the source variable already for me, so this regex was meant to detect it inside the original message, including the "=tex" part, which would be at the start. I have modified it to work with that now.